AutoRest/TestServer/swagger/paging.json

```diff
@@ -18,7 +18,7 @@
   "paths": {
     "/paging/single": {
       "get": {
-        "x-ms-pageable": { "nextLinkName": "nextLink", "itemName": "values" },
+        "x-ms-pageable": { "nextLinkName": "next.Link", "itemName": "values" },
```

Comment:
@xingwu1 Could the '.' scenario be tested in its own scenario in the test server? It makes it harder to know when this specific behavior breaks if it's part of an existing paging test. I would prefer to see it as a test in a new path in a swagger file, not a modification of an existing one.